packages/ds/src/admin/ui/Tooltip.tsx

```diff
@@ -19,7 +20,16 @@ export default function Tooltip({
   className,
   tooltipClassName,
 }: TooltipProps) {
-  const tooltipId = `tooltip-${id}`;
+  const tooltipId = id ? `tooltip-${id}` : generateSlugHashId(title);
+
+  function generateSlugHashId(input: string): string {
+    const utf8Bytes = new TextEncoder().encode(input);
+    const base64 = btoa(String.fromCharCode(...utf8Bytes));
+
+    const normalized = slugify(base64, { lower: true });
```

Comment:
```suggestion
    const normalized = input.normalize("NFD")
      .replace(/[\u0300-\u036f]/g, "") //remove accents
      .replaceAll(" ", "_") // remove spaces
     .replace(/[^\/a-zA-Z0-9._-]/g, ""); // remove special characters
```
ça me parait bien compliqué pour pas grand chose, que pense tu de ça plutôt ? (ça permet notamment de dépendances externes)